README.md

```diff
@@ -55,10 +55,15 @@ To run all unit tests, you can use : `npm run test`.
 [More information about unit tests.](/tests/README.md)
 
 ### Compile Whole Dictionary
+1.  Install dependencies via npm. 
 
 `npm run compile`
```

Comment:
you can remove this as it's not part of step1. or even replace it with `npm i`

Reply:
copy pasta error lol thats what i meant to put

Reply:
🍝